main.star

```diff
@@ -234,6 +235,18 @@ def run(plan, args={}):
                 network_params.genesis_delay,
             )
             plan.print("Succesfully launched blob spammer")
+        elif additional_service == "goomy_blob":
```

Comment:
we could also do this the way we do dora / full explorer so via `explorer_version`; we can add a `blob_explorer_type` perhaps

Reply:
as discussed on discord, we'd like to proceed with the additional_services array